index.d.ts

```diff
@@ -76,7 +76,26 @@ declare namespace activeWin {
 		platform: 'windows';
 	}
 
+	interface BaseAccessResult {
+		all: boolean;
+	}
+
+	interface MacOSAccessResult extends BaseAccessResult {
+		screen: boolean;
+		accessibility: boolean;
+		platform: 'macos';
+	}
+
+	interface WindowsAccessResult extends BaseAccessResult {
+		platform: 'windows';
+	}
+
+	interface LinuxAccessResult extends BaseAccessResult {
+		platform: 'linux';
+	}
```

Comment:
I don't see the value in complicating this. We can just always return those properties, and if Windows/Linux, we should return `true` for them and document that they will always be `true`.